hc/accounts/tests/test_check_token.py

```diff
@@ -22,7 +22,17 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_it_redirects_when_logged_in(self):
+        self.client.login(username="alice@example.org", password="password")
```

Comment:
I see we are inheriting from `BaseTestCase` and in the `setup()` we call super which calls the `setup()` method of `BaseTestCase` and already creates alice (self.alice)
So for `username` we can do self.alice.email